test_data/definitions.yaml

```diff
@@ -24,9 +24,11 @@ urls: ['http://support.microsoft.com/kb//214653']
 ---
 name: CurrentControlSet
 doc: The control set the system is currently using.
-collectors:
-- collector_type: REGISTRY_VALUE
-  args: {path_list: ['HKEY_LOCAL_MACHINE\SYSTEM\Select\Current']}
+sources:
+- type: REGISTRY_VALUE
+  attributes:
+    key_value_pairs:
+      - {key: 'HKEY_LOCAL_MACHINE\SYSTEM\Select', value: 'Current'}
```

Comment:
Yeah, you're right, always returning the default value for any matching key is probably fine.